app/src/main/java/org/fossasia/openevent/utils/KeyBoardSupport.java

```diff
@@ -0,0 +1,35 @@
+package org.fossasia.openevent.utils;
+
+import android.app.Activity;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.EditText;
+
+/**
+ * User : opticod(Anupam Das)
+ * Date : 16/2/16.
+ */
+public class KeyBoardSupport {
+    public static void helperHideKeyboard(View view, final Activity activity) {
```

Comment:
if your function is called `helperHideKeyboard ` then why is it putting a touch listener on individual views?